src/amo/pages/AddonReviewList/index.js

```diff
@@ -51,6 +51,10 @@ import Notice from 'ui/components/Notice';
 
 import './styles.scss';
 
+function getCurrentPage(location: ReactRouterLocationType) {
+  return location.query.page || '1';
+}
```

Comment:
Could we adopt fat arrow functions everywhere?

Reply:
Why? They require more characters to type. I mean, not that I'm bothered by that but I don't see why it would be helpful to use them everywhere. Just so we don't have to decide which one to use?

Reply:
> Just so we don't have to decide which one to use?

Yes.

> They require more characters to type.

60 vs. 63? 😹 

I'd argue that you can make this function shorter then, saving lots of chars:

```js
const getCurrentPage = (location: ReactRouterLocationType) => location.query.page || '1';
```

No really, consistency would be nice, especially since we already discussed that point multiple times 😇